quickstarts/microsoft.documentdb/cosmosdb-cassandra-autoscale/main.bicep

```diff
@@ -0,0 +1,163 @@
+@description('Cosmos DB account name, max length 44 characters')
+param accountName string = 'cassandra-${uniqueString(resourceGroup().id)}'
+
+@description('Location for the Cosmos DB account.')
+param location string = resourceGroup().location
+
+@description('The primary replica region for the Cosmos DB account.')
+param primaryRegion string
+
+@description('The secondary replica region for the Cosmos DB account.')
+param secondaryRegion string
+
+@allowed([
+  'Eventual'
+  'ConsistentPrefix'
+  'Session'
+  'BoundedStaleness'
+  'Strong'
+])
+@description('The default consistency level of the Cosmos DB account.')
+param defaultConsistencyLevel string = 'Session'
+
+@minValue(10)
+@maxValue(1000000)
+@description('Max stale requests. Required for BoundedStaleness. Valid ranges, Single Region: 10 to 1000000. Multi Region: 100000 to 1000000.')
+param maxStalenessPrefix int = 100000
+
+@minValue(5)
+@maxValue(86400)
+@description('Max lag time (seconds). Required for BoundedStaleness. Valid ranges, Single Region: 5 to 84600. Multi Region: 300 to 86400.')
+param maxIntervalInSeconds int = 300
+
+@allowed([
```

Comment:
ok, my guess is you'll have this on all of the sample, do a scrub (I won't comment on the rest) and lmk when you're done...